sdks/python/apache_beam/runners/direct/executor.py

```diff
@@ -278,12 +280,11 @@ def __init__(self, transform_evaluator_registry, evaluation_context,
     self._retry_count = 0
     # Switch to turn on/off the retry of bundles.
     pipeline_options = self._evaluation_context.pipeline_options
+    # TODO(mariagh): Remove if once "bundle retry" is no longer experimental.
```

Comment:
<!--new_thread; commit:593b6f6659973b2774659a66e9be613c535c3f16; resolved:0-->
"Remove once"

Reply:
I meant to say Remove if-statement once...